package net.mystoria.framework.authentication.exception

class AuthenticationFailedException : RuntimeException("There was an error trying to log you in.") {
}